zhaquirks/legrand/dimmer.py

```diff
@@ -93,3 +93,42 @@ class DimmerWithoutNeutral(CustomDevice):
             }
         }
     }
+
+
+class DimmerWithoutNeutral2(DimmerWithoutNeutral):
+    """Dimmer switch w/o neutral 2."""
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=256
+        # device_version=1
+        # input_clusters=[0, 3, 4, 8, 6, 5, 15, 64513]
+        # output_clusters=[0, 64513, 25]>
+        MODELS_INFO: [(f" {LEGRAND}", " Dimmer switch w/o neutral")],
```

Comment:
are they all with space in front? if yes, maybe just update the constant?

Reply:
I don’t know. We only have this device so far.

Reply:
So the question if they ever going to  fix it.

Reply:
The two other Legrand device I tested so far do have a leading space too:

https://github.com/zigpy/zha-device-handlers/issues/299#issuecomment-598973755

https://github.com/zigpy/zha-device-handlers/issues/299#issuecomment-598847708